### PR TITLE
ERC Proxy Identity draft

### PR DESCRIPTION
**Pull request by frozeman**
This PR adds the ERC identity draft with slight modifications to the discussed #725.
Once this PR is merged we can discuss specific changes by making separate PRs to the draft itself.

Following modifications are present: 

I removed the `getKeyPurposes` in favour of `keyHasPurpose`, which returns a bool and will work as a check for checking keys at an identity.

I also changed the key structure to keep an array of purpose IDs in the struct of the key, but we should discuss the idea of @nick to use a bit mask instead for storing purposes later.

### CODE REVIEW

File: EIPS/eip-X-identity.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
```
Comment (Arachnid) [inline EIPS/eip-X-identity.md]: Please rename to eip-725.md and number 725.

File: EIPS/eip-X-identity.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+title: ERC-725 Identity
```
Comment (Arachnid) [inline EIPS/eip-X-identity.md]: No need to include the EIP/ERC number in the title.

Comment (nicksavers) [reply EIPS/eip-X-identity.md]: I would also argue that the title is a bit broad.

File: EIPS/eip-X-identity.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+author: Fabian Vogelsteller (@frozeman) <fabian@ethereum.org>
```
Comment (nicksavers) [inline EIPS/eip-X-identity.md]: I don't think the build script will know how to parse this into a single link.
Either GitHub username **or** e-mail address should be sufficient.
See: https://github.com/ethereum/EIPs/blob/master/_includes/authorlist.html

File: EIPS/eip-X-identity.md (hunk 2 above)
Comment (frozeman) [reply EIPS/eip-X-identity.md]: I renamed it to Proxy Identity to be more specific

**Comment by Arachnid** [comment]
Please rename to `eip-725.md`.

**Comment by Arachnid** [comment]
Also, see the build failure-  you have an invalid internal link in your document.

**Comment by frozeman** [comment]
Why not adding the title in the md document name, like we have for ERC20: `eip-20-token-standard.md`?
Also your drafts have a title added?

**Comment by Arachnid** [comment]
@frozeman Because that's not the naming convention we use for merged drafts.

**Pull request merged**